applications/hotornot/src/hotornot_maintenance.erl

```diff
@@ -24,12 +24,12 @@
 local_summary() ->
     io:format("use rates_for_did/1 to see what rates would be used for a DID").
 
--spec rates_for_did(ne_binary()) -> 'ok'.
--spec rates_for_did(ne_binary(), api_binary(), trunking_options()) -> 'ok'.
-rates_for_did(DID) ->
-    rates_for_did(DID, 'undefined', []).
-rates_for_did(DID, Direction, RouteOptions) when is_list(RouteOptions) ->
-    case hon_util:candidate_rates(DID) of
+-spec rates_for_did(ne_binary(), ne_binary(), api_binary()) -> 'ok'.
+-spec rates_for_did(ne_binary(), ne_binary(), api_binary(), trunking_options()) -> 'ok'.
+rates_for_did(AccountId, DID, Direction) ->
```

Comment:
We should keep arity-2 for querying the global ratedeck since that's what most people have.